book_source/basic_users_guide/Choosing-soils.Rmd

```diff
@@ -8,32 +8,11 @@ As with [[Choosing initial vegetation]], we eventually hope to develop data stan
 
 A PEcAn-standard netCDF file format exists for soil texture, depth, and physical parameters, using PEcAn standard names that are largely a direct extention of the CF standard. A table of standard names and units can be listed using ```PEcAn.data.land::soil.units()``` with no arguements.
 
-```{r}
+```{r,echo=FALSE}
 knitr::kable(PEcAn.data.land::soil.units())
 ```
 
-
-Local data that has the correct names and units can easily be written out in PEcAn standard using the function soil2netcdf.
```

Comment:
Not sure I like the decision to move this info out of this section and into the "how to make an input converter" section. That section was intended to be part of the _DEVELOPERS_ guide explaining how to build conversion functions. The recent decision to move a lot of info on how to _USE_ modules out of _USERS_ guide and into the developers section muddles what I thought was a clear line, and leaves the resulting text a bit confused -- I suspect that if I didn't know the system that I'd find it hard to figure out what I was actually supposed to do to use get runs started.